papers/2402.07712_Model_Collapse_Demystified:_The_Case_of_Regression.md

```diff
@@ -0,0 +1,61 @@
+# [Model Collapse Demystified: The Case of Regression](https://arxiv.org/abs/2402.07712)
+
+ID: Issue #246
+
+## Summary
+
+Model collapse is when a model is trained recursively on data generated from previous generations of itself over time, and its performance degrades until the model eventually becomes completely useless. 
+This paper is a theoretical study of model collapse in the setting of high-dimensional regression. 
+They obtain analytic formulae which quantitatively outline this phenomenon (linear regression with extension to kernel methods). 
+They assume complete data replacement at each epoch. 
+They also propose a simple strategy based on adaptive regularization to mitigate model collapse.
+
+## Relevance to survey topic (1-5)
+
+Relevance: 4
```

Comment:
Just for your own calibration, I'd say this is probably a bit less relevant to us (maybe a 2.5)